plugins/modules/user_role_workflow_manager.py

```diff
@@ -1117,9 +1117,11 @@ def valid_user_config_parameters(self, user_config):
         if password:
             self.validate_password(password, error_messages)
 
+        username_regex = re.compile(r"^[A-Za-z0-9@._-]{3,50}$")
+        username_regex_msg = "The username cannot contain any special characters and must be 3 to 50 characters long."
```

Comment:
username_regex_msg = "The username must not contain any special characters and must be 3 to 50 characters long."

Reply:
Updated the code accordingly.